oe_theme.theme

```diff
@@ -413,6 +413,30 @@ function oe_theme_preprocess_pattern_accordion(&$variables) {
   }
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
```

Comment:
Comment here is wrong: `template_preprocess_pattern_dropdown()`

Reply:
Can you take a look on the others as well?